package.json

```diff
@@ -39,41 +30,34 @@
 			"name": "Patrick De Young",
 			"email": "hello@patrickdeyoung.me",
 			"url": "https://patrickdeyoung.me"
+		},
+		{
+			"name": "Sukhraj Ghuman",
+			"email": "sukhrajghuman@live.com",
+			"url": "https://github.com/sukhrajghuman"
 		}
 	],
 	"license": "MIT",
 	"devDependencies": {
-		"browser-sync": "^2.18.8",
-		"onchange": "^3.2.1"
+		"autoprefixer": "^9.0.1",
```

Comment:
`autoprefixer`, `node-sass`, `postcss-cli` are all dependencies. Otherwise we will never have a css on the page.